app/src/main/res/layout/activity_sign_in.xml

```diff
@@ -2,103 +2,166 @@
+        android:layout_height="match_parent"
+        android:contentDescription="@string/background_image"
         android:scaleType="fitXY"
+        android:src="@drawable/splash_screen_background"
         android:visibility="visible"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
-        app:srcCompat="@drawable/sign_in_background"
-        android:contentDescription="@string/background_image" />
+        app:layout_constraintTop_toTopOf="parent" />
+
+    <View
+        android:layout_width="match_parent"
+        android:layout_height="match_parent"
+        android:background="@color/secondaryTransparent"></View>
 
     <ImageView
         android:id="@+id/logo"
-        android:layout_width="@dimen/sign_in_logo_size"
-        android:layout_height="@dimen/sign_in_logo_size"
-        android:layout_marginStart="@dimen/sign_in_logo_margin"
-        android:layout_marginTop="@dimen/sign_in_logo_margin"
-        android:layout_marginEnd="@dimen/sign_in_logo_margin"
-        android:layout_marginBottom="@dimen/sign_in_logo_margin"
-        app:layout_constraintBottom_toBottomOf="@+id/background"
-        app:layout_constraintEnd_toEndOf="@+id/background"
-        app:layout_constraintStart_toStartOf="@+id/background"
-        app:layout_constraintTop_toTopOf="@+id/background"
-        app:srcCompat="@drawable/endava_logo"
-        android:contentDescription="@string/sign_in_logo" />
-
-    <EditText
-        android:id="@+id/email"
-        android:layout_width="0dp"
-        android:text="gabriel.blaj@endava.com"
-        android:layout_height="wrap_content"
-        android:layout_marginStart="@dimen/sign_in_email_horizontal_margins"
-        android:layout_marginTop="@dimen/sign_in_email_top_margin"
-        android:layout_marginEnd="@dimen/sign_in_email_horizontal_margins"
-        android:drawableStart="@drawable/ic_email_orange"
-        android:drawablePadding="@dimen/sign_in_input_drawable_padding"
-        android:ems="10"
-        android:hint="@string/email_hint"
-        android:inputType="textEmailAddress"
+        android:layout_width="@dimen/sign_in_logo_width"
+        android:layout_height="@dimen/sign_in_logo_height"
+        android:layout_marginLeft="@dimen/sign_in_logo_margin_left"
+        android:layout_marginRight="@dimen/sign_in_logo_margin_right"
+        android:layout_marginBottom="@dimen/sign_in_logo_margin_bottom"
+        android:contentDescription="@string/sign_in_logo"
+        app:layout_constraintBottom_toTopOf="@id/info_container"
+        app:layout_constraintEnd_toEndOf="@id/info_container"
+        app:layout_constraintStart_toStartOf="@id/info_container"
+        app:srcCompat="@drawable/logo_endava" />
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/info_container"
+        android:layout_width="@dimen/info_container_width"
```

Comment:
Use match_constraints(0dp) so it respects the left margin and does not overlay outside the screen on specific devices